guides/source/action_text_overview.md

```diff
@@ -45,9 +45,31 @@ happens after every keystroke, and avoids the need to use execCommand at all.
 
 ## Installation
 
-Run `rails action_text:install` to add the Yarn package and copy over the necessary migration.
-Also, you need to set up Active Storage for embedded images and other attachments.
-Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+Run `rails action_text:install` to add the Yarn package and copy over the necessary migration. Also, you need to set up Active Storage for embedded images and other attachments. Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+
+### Installation with Webpacker 
```

Comment:
Now those instructions will be confusion for people that have run the installer and it was successful. They will not know if they should follow those steps or not. Can you clarify?

Reply:
I see your point. Zooming out, the issue I'm trying to solve for is when the installer fails, as happened in my case (I store my webpacker stuff in `app/webpacker`). I'd like for the docs to have a fallback reference.

Perhaps I could relabel the new section "Troubleshooting the Installation"?

Reply:
I imagine that was the problem. We try to focus on explaining what the installer will do, and maybe avoid the new subsection.

Something like:

> Run `rails action_text:install` to add the Yarn package and copy over the necessary migration. The installer will...

Reply:
OK I've reframed in terms of what happen as you suggested.